asmdiff.sh

```diff
@@ -3,4 +3,4 @@
 OBJDUMP="$DEVKITPPC/bin/powerpc-eabi-objdump -D -bbinary -EB -mpowerpc -M gekko --adjust-vma=0x80003200"
 !(test -f baserom.dump) && $OBJDUMP baserom.dol > baserom.dump
 !(test -f main.dump) && $OBJDUMP main.dol > main.dump
-diff -u --color=always baserom.dump main.dump | head -n 2000
\ No newline at end of file
+diff -u --color=always baserom.dump main.dump | head -n 3000
```

Comment:
why change this?

Reply:
Personal preference; again, forgot to remove it from the commit.